src/main/java/org/opentripplanner/middleware/models/ItineraryExistence.java

```diff
@@ -186,7 +223,8 @@ public void checkExistence() {
                 setResultForDayOfWeek(result, dayOfWeek);
             }
             // Send off each plan query to OTP.
-            OtpDispatcherResponse response = OtpDispatcher.sendOtpPlanRequest(OtpVersion.OTP1, otpRequest);
+            String variables = ItineraryExistence.paramsToVariables(otpRequest.requestParameters);
+            OtpDispatcherResponse response = OtpDispatcher.sendGraphQLPostRequest(OtpVersion.OTP2, variables);
             TripPlan plan = null;
             try {
                 plan = response.getResponse().plan;
```

Comment:
for OTP2, it is `response.getResponse().data.plan`. I think we need to create a new class `Otp2Response` to reflect that.